week-03/main.js

```diff
@@ -3,6 +3,8 @@
 // Given the following array
 var list = [1, 2, 3, 4, 5];
 // Remove one from the beginning of the array and add it to the end
+list.shift(1);
```

Comment:
The shift() method removes the first element from an array and returns that element.   It does not take any arguments.
Push adds an element. However you're not actually pushing in the element that was shifted out.